core/db.py

```
@@ -6,13 +6,37 @@
 
 from __future__ import annotations
 
+import asyncio
```

Flake8 (7.3.0)

[error] 'asyncio' imported but unused

(F401)